api/rda.py

```diff
@@ -36,24 +36,22 @@ def wrapper(body, **kwargs):
         # Get the identifiers through a search query
         ids = [item_id]
 
-        # FIXME oai-pmh should be no different
         downstream_logger = evaluator.logger
```

Comment:
`downstream_logger = None` (`evaluator` is not set)

Reply:
Done